packages/api/src/perma-cache/post.js

```diff
@@ -0,0 +1,238 @@
+/* eslint-env serviceworker, browser */
+/* global Response */
+
+import { FAR_FUTURE, PAD_LEN, MAX_ALLOWED_URL_LENGTH } from '../constants.js'
```

Comment:
For `FAR_FUTURE` and `PAD_LEN`, personally I'd move them in this file, or give them better names.

Reply:
This will be used in more than post, so created `utils.js` within perma-cache scope.